### PR TITLE
migrate to ce3

### PR DESCRIPTION
**Pull request by amalinkoevo**

### CODE REVIEW

**Comment by olegkovalenko** [comment]
lets update version to 2.0.0-SNAPSHOT as 2.0.0 will be automatically set by sbt-release during release process

**Comment by olegkovalenko** [comment]
Can you please also drop a couple of lines in README.md saying that the library is based on CE3 starting from 2.0.0 and update the version in the "Setup" section?

File: project/Dependencies.scala [hunk 1]
```diff
@@ -4,14 +4,15 @@ object Dependencies {
...
   val scalatest        = "org.scalatest"       %% "scalatest"      % "3.2.9"
   val `executor-tools` = "com.evolutiongaming" %% "executor-tools" % "1.0.2"
-  val `cats-helper`    = "com.evolutiongaming" %% "cats-helper"    % "2.7.2"
-  val `ddata-tools`    = "com.evolutiongaming" %% "ddata-tools"    % "2.0.8"
+  val `cats-helper`    = "com.evolutiongaming" %% "cats-helper"    % "3.0.3"
+  val `ddata-tools`    = "com.evolutiongaming" %% "ddata-tools"    % "3.0.0"
   val `safe-actor`     = "com.evolutiongaming" %% "safe-actor"     % "3.0.0"
 
   object Cats {
     private val version = "2.3.0"
```
Comment (olegkovalenko) [inline project/Dependencies.scala:12]: shall we update it to 2.7.0 ?

Comment (amalinkoevo) [reply project/Dependencies.scala]: Done

**Comment by olegkovalenko** [comment]
LGTM 👍 

@Z1kkurat wdyt ?

**Comment by Z1kkurat** [comment]
@amalinkoevo 2.12 build is failing, please check what's going on there

**Comment by olegkovalenko** [comment]
pushed series/1.x.x branch for cats-effect 2 back ports

**Comment by amalinkoevo** [comment]
@Z1kkurat compilation for 2.12 is fixed. Compilation failed because idea shows import scala.collection.immutable.IndexedSeq as unused, but actually it is used for 2.12

**Review by Z1kkurat — APPROVED** [review]

**Pull request merged**